docs/source/topics/proc_connecting-to-remote-instance.adoc

```diff
@@ -0,0 +1,63 @@
+[id="connecting-to-remote-instance_{context}"]
+= Connecting to a remote {prod} instance
+
+Follow this procedure to connect a client machine to a remote server running a {prod} OpenShift cluster.
```

Comment:
Should we somewhere mention that client and remote machine is considered Fedora/CentOS/RHEL based because all our doc commands are based on those platform.